src/core/Akka/IO/TcpConnection.cs

```diff
@@ -945,6 +945,7 @@ public override void DoWrite(ConnectionInfo info)
             {
                 try
                 {
+                    // TODO: avoid use of SocketAsyncEventArgs on newer platforms
```

Comment:
Stupid question, are newer APIs to the point where SAEA isn't worth the squeeze? Again it's been a while but last I knew SAEA was still the 'best' way.